test/rubygems/test_gem_commands_build_command.rb

```diff
@@ -230,6 +230,90 @@ def test_execute_outside_dir
     assert_equal "this is a summary", spec.summary
   end
 
+  def test_execute_current_dir
+    gem_0 = util_spec 'some_gem' do |s|
+      s.license = 'AGPL-3.0'
+      s.files = ['README.md']
+    end
+
+    gem_1 = util_spec 'another_gem' do |s|
+      s.license = 'AGPL-3.0'
+      s.files = ['README.md']
+    end
```

Comment:
My fear is that supporting this might cause confusion and it's hard to think of cases where it's useful. For example, in this test we're defining two different gems that have the same README... right? :thinking: